scripts/generate-dsl-info.mjs

```diff
@@ -0,0 +1,174 @@
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import fetch from "node-fetch";
+
+// API endpoints
+const NETWORK_CONFIG_ENDPOINT =
+	process.env.NETWORK_CONFIG_ENDPOINT || "https://gi.rss3.dev";
```

Comment:
I don't think we need different endpoints based on env here? Can just fetch from production until we need development stage docs in the future.